java-client/src/main/java/com/cumulocity/sdk/client/RestOperations.java

```diff
@@ -30,6 +30,8 @@ <T extends ResourceRepresentation> T putStream(String path, String contentType,
     <T extends ResourceRepresentation> T putStream(String path, MediaType mediaType, InputStream content,
                                                    Class<T> responseClass);
 
+    void postStreamWithoutResponse(String path, InputStream inputStream, MediaType inputStreamMediaType);
```

Comment:
I'm not huge fan of such names, postStream() should be fine, "withoutRresponse" is visible from method signature :)

Reply:
yee I would also didn't thought about such naming but I just blindly follow the convention which was already in the file. Maybe it doesn't make sense to multiply bad practice.
Here you go sir - #350